resolwe/flow/tests/test_commands.py

```diff
@@ -88,3 +91,53 @@ def test_process_register_no_admin(self):
         err = StringIO()
         self.assertRaises(SystemExit, call_command, 'register', path=[PROCESSES_DIR], stderr=err)
         self.assertEqual('Admin does not exist: create a superuser\n', err.getvalue())
+
+
+class ListDockerImagesTest(ProcessTestCase):
```

Comment:
No, above tests are testing `register` function, so it has to be used. 

But in general reading and parsing yaml files is very slow (especially if you do it in each test), so `_register_schemas` only do it once and then use cached results for the rest of the test case.